addons/account/wizard/account_tour_upload_bill.py

```diff
@@ -147,3 +161,7 @@ def apply(self):
         purchase_journal = self.env['account.journal'].search([('type', '=', 'purchase')], limit=1)
         bill_ids = self.env['account.move'].search([('journal_id', '=', purchase_journal.id)]).ids
         return self.env['account.tour.upload.bill']._action_list_view_bill(bill_ids)
+
+class AccountTourUploadBillEmailNoAlias(models.TransientModel):
```

Comment:
i find it weird to add an empty class, can't we just add some logic in the xml template when there is no alias ?